<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>MainWindow</class>
  <widget class="QMainWindow" name="MainWindow">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>839</width>
        <height>610</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>MainWindow</string>
    </property>
    <widget class="QWidget" name="centralWidget">
      <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
        <bool>true</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
        <item>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0,0">
            <property name="spacing">
              <number>5</number>
            </property>
            <property name="sizeConstraint">
              <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <item>
              <widget class="QPushButton" name="ConnectButton">
                <property name="minimumSize">
                  <size>
                    <width>130</width>
                    <height>30</height>
                  </size>
                </property>
                <property name="maximumSize">
                  <size>
                    <width>130</width>
                    <height>30</height>
                  </size>
                </property>
                <property name="text">
                  <string>Connect</string>
                </property>
                <property name="shortcut">
                  <string>Alt+C</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QPushButton" name="DisconnectButton">
                <property name="enabled">
                  <bool>false</bool>
                </property>
                <property name="minimumSize">
                  <size>
                    <width>130</width>
                    <height>30</height>
                  </size>
                </property>
                <property name="maximumSize">
                  <size>
                    <width>120</width>
                    <height>30</height>
                  </size>
                </property>
                <property name="text">
                  <string>Disconnect</string>
                </property>
                <property name="shortcut">
                  <string>Alt+D</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QPushButton" name="ProcessButton">
                <property name="enabled">
                  <bool>false</bool>
                </property>
                <property name="minimumSize">
                  <size>
                    <width>130</width>
                    <height>30</height>
                  </size>
                </property>
                <property name="maximumSize">
                  <size>
                    <width>120</width>
                    <height>30</height>
                  </size>
                </property>
                <property name="text">
                  <string>Process</string>
                </property>
                <property name="shortcut">
                  <string>F3</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QPushButton" name="CalibrateButton">
                <property name="enabled">
                  <bool>false</bool>
                </property>
                <property name="minimumSize">
                  <size>
                    <width>130</width>
                    <height>30</height>
                  </size>
                </property>
                <property name="maximumSize">
                  <size>
                    <width>130</width>
                    <height>30</height>
                  </size>
                </property>
                <property name="text">
                  <string>Calibrate</string>
                </property>
                <property name="shortcut">
                  <string>Ctrl+R</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QCheckBox" name="VdCheckBox">
                <property name="text">
                  <string>VD enable</string>
                </property>
              </widget>
            </item>
            <item>
              <spacer name="horizontalSpacer">
                <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                  <size>
                    <width>40</width>
                    <height>20</height>
                  </size>
                </property>
              </spacer>
            </item>
          </layout>
        </item>
        <item>
          <widget class="QCheckBox" name="JsonCheckBox">
            <property name="text">
              <string>Save as JSON</string>
            </property>
            <property name="checked">
              <bool>true</bool>
            </property>
          </widget>
        </item>
        <item>
          <widget class="QTabWidget" name="tabWidget" />
        </item>
      </layout>
    </widget>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <resources />
  <connections />
</ui>
